tests/test_core.py

```diff
@@ -442,7 +442,13 @@ def test_sintvars(self):
 
   def test_int53(self):
     self.emcc_args += ['-sDEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[$convertI32PairToI53,$convertU32PairToI53,$readI53FromU64,$readI53FromI64,$writeI53ToI64,$writeI53ToI64Clamped,$writeI53ToU64Clamped,$writeI53ToI64Signaling,$writeI53ToU64Signaling]']
-    self.do_core_test('test_int53.c', interleaved_output=False)
+   
+    if EMTEST_REBASELINE:
+      self.run_process([EMCC, test_file('core/test_int53.c'), '-o', Path('a.js'), '-DGENERATE_ANSWERS'] + self.emcc_args)
+      ret = self.run_process(config.NODE_JS + [Path('a.js')], stdout=PIPE).stdout
+      open(test_file('core/test_int53.out'), 'w').write(ret)
```

Comment:
`self.create_file(test_file('core/test_int53.out'), ret)`

Reply:
That does not seem to work, but fails with an `assert not name.is_absolute()`.